lms/djangoapps/discussion_api/views.py

```diff
@@ -44,23 +44,29 @@ class CourseView(_ViewMixin, DeveloperErrorViewMixin, APIView):
 
     **Example Requests**:
 
-        GET /api/discussion/v1/courses/course-v1:ExampleX+Subject101+2015
+        GET /api/discussion/v1/courses/{course_id}
 
     **Response Values**:
 
-        * id: The identifier of the course
+        * id: The unique identifier for the course.
 
-        * blackouts: A list of objects representing blackout periods (during
-            which discussions are read-only except for privileged users). Each
-            item in the list includes:
+        * blackouts: A list of objects that represent any defined blackout
+          periods during the course run. Discussions are read-only during a
+          blackout period, except for users who have privileged discussion
+          moderation roles in the course. Each item in the list includes the
+          following values.
 
-            * start: The ISO 8601 timestamp for the start of the blackout period
+            * start: The ISO 8601 timestamp for the start of the blackout
+              period.
 
-            * end: The ISO 8601 timestamp for the end of the blackout period
+            * end: The ISO 8601 timestamp for the end of the blackout period.
 
-        * thread_list_url: The URL of the list of all threads in the course.
+        * following_thread_list_url: TBD
```

Comment:
From the API proposal: 

"following_thread_list_url: String. The URL of the list of the user's followed threads in the course"